doc/whats-new.rst

```diff
@@ -131,7 +131,11 @@ Bug fixes
   ``rtol`` arguments when called on ``DataArray`` objects.
   By `Stephan Hoyer <https://github.com/shoyer>`_.
 
-- Xarray ``quantile`` methods now properly raise a ``TypeError`` when applied to
+- Stop ``repr`` and the Jupyter Notebook from automatically computing dask
```

Comment:
This should probably go under "breaking changes" instead, given that it changes existing behavior.